src/protocols/screencopy.rs

```diff
@@ -181,23 +201,21 @@ where
 
         // Send desired SHM buffer parameters.
         frame.buffer(
-            wl_shm::Format::Argb8888,
+            wl_shm::Format::Xrgb8888,
             buffer_size.w as u32,
             buffer_size.h as u32,
             buffer_size.w as u32 * 4,
         );
 
-        // if manager.version() >= 3 {
-        //     // Send desired DMA buffer parameters.
-        //     frame.linux_dmabuf(
-        //         Fourcc::Argb8888 as u32,
-        //         buffer_size.w as u32,
-        //         buffer_size.h as u32,
-        //     );
-        //
-        //     // Notify client that all supported buffers were enumerated.
-        //     frame.buffer_done();
-        // }
+        // Send desired DMA buffer parameters.
+        frame.linux_dmabuf(
+            Fourcc::Xrgb8888 as u32,
+            buffer_size.w as u32,
+            buffer_size.h as u32,
```

Comment:
When an output resizes, or the output transform changes, or the output disconnects, it should destroy the screencopy objects for this output.

Reply:
Right now we are checking for output existence but not the other changes. Do you mean destroying the screencopy manager as well?

Reply:
Might be worth checking what sway does. Because I'm not sure what `damage` you're supposed to send after an output resize. And destroying the screencopy manager solves that problem. Though, if sway doesn't destroy the manager, then clients won't expect it.